bridge/server/api/index.js

```diff
@@ -119,5 +54,23 @@ module.exports = (params) => {
     }
   });
 
+  router.all('*', async (req, res, next) => {
+    try {
+      const result = await axios({
+        method: req.method,
+        url: `${apiUrl}${req.url}`,
+        data: req.params,
```

Comment:
are we sure we won't run into the same problem here as with the version check with CloudFront?